src/components/Box/Box.tsx

```diff
@@ -62,6 +63,11 @@ export const useStyles = makeStyles<BoxProps>(
       if (typeof value === 'number') return theme.spacing(value);
     };
 
+    const stringOrThemeSpacingGap = (value: string | number | undefined) => {
+      if (typeof value === 'string') return value;
+      if (typeof value === 'number') return theme.spacing(value) + 'px';
```

Comment:
I'm not too familiar with this so sorry if I am wrong, but doesn't `theme.spacing` already return the value as pixels?

Reply:
This confused me as well. Using `+ px` was the only way I could get `gap` to work with strings and spacing units. I'm not really sure what the problem is. I can tell you I've had to add `px` when using `gap` with `theme.spacing` in other Lifeology repos as well. 

It looks like MUI had this same problem at one point: https://github.com/mui-org/material-ui/issues/24773

Reply:
Ah interesting, yeah, seems like `px` is required here for this: https://github.com/mui-org/material-ui/pull/24794/files#diff-c66b437e5d952a0998e624e32687b4f3114020624f2fee56d4894f27579874f3R281

> Use spacing unit in `gap`, `rowGap`, and `columnGap`. If you were using a number previously, you need to mention the px to bypass the new transformation with `theme.spacing`.

Reply:
would `theme.pxToRem` be a better solution than concatenating `px`?